third_party/musl/src/starboard/errno/__errno_location.c

```diff
@@ -0,0 +1,39 @@
+#include <errno.h>
```

Comment:
I think we are missing Cobalt copyright header here. Probably everything under musl/src/starboard should have it.

But if those weren't here before it's fine to submit as is